client/checkout/classic/event-handlers.js

```diff
@@ -176,13 +176,39 @@ jQuery( function ( $ ) {
 				}
 
 				if ( targetLabel ) {
+					// wrapInner target label in a span.woopayments-inner-label if it's not already
+					let targetLabelInnerSpan = targetLabel.querySelector(
+						'span.woopayments-inner-label'
+					);
+					if ( ! targetLabelInnerSpan ) {
+						const targetLabelInner = targetLabel.innerHTML;
+						targetLabel.innerHTML = '';
+						targetLabelInnerSpan = document.createElement( 'span' );
+						targetLabelInnerSpan.classList.add(
+							'woopayments-inner-label'
+						);
+						targetLabelInnerSpan.innerHTML = targetLabelInner;
+						targetLabel.appendChild( targetLabelInnerSpan );
+					}
+
+					let spacer = targetLabel.querySelector( 'span.spacer' );
```

Comment:
I think it would be helpful for our future selves to add a comment here to indicate that the spacer has been added because WC Core adds the `&nbsp;`, and the spacer is useful to ensure all labels are vertically aligned properly